Code/GraphMol/FileParsers/PDBParser.cpp

```diff
@@ -543,8 +560,10 @@ RWMol *PDBBlockToMol(const char *str, bool sanitize, bool removeHs,
 
   if (!mol) return (RWMol *)0;
 
-  ConnectTheDots(mol, ctdIGNORE_H_H_CONTACTS);
-  StandardPDBResidueBondOrders(mol);
+  if (proximityBonding) {
+    ConnectTheDots(mol, ctdIGNORE_H_H_CONTACTS);
+    StandardPDBResidueBondOrders(mol);
```

Comment:
I was going to say no, but if you had a PDB file in "flavor" that don't encode bonds multiple times (`flavor & 8 ` as defined by Roger) then you could use `StandardPDBResidueBondOrders`. On the other hand I see the `proximityBonding` as a "don't touch bonding at all" flag. This is particularly useful when you've edited the molecule so that it sanitizes and you want to read it back from a "clean" PDB.